src/main/java/com/stripe/net/OAuth.java

```diff
@@ -63,10 +68,15 @@ public static DeauthorizedAccount deauthorize(Map<String, Object> params, Reques
     Map<String, Object> paramsCopy = new HashMap<>();
     paramsCopy.putAll(params);
 
-    String url = ApiResource.fullUrl(Stripe.getConnectBase(), options, "/oauth/deauthorize");
     paramsCopy.put("client_id", getClientId(paramsCopy, options));
-    return OAuth.stripeResponseGetter.oauthRequest(
-        ApiResource.RequestMethod.POST, url, paramsCopy, DeauthorizedAccount.class, options);
+    return OAuth.globalResponseGetter.request(
+        BaseAddress.CONNECT,
+        ApiResource.RequestMethod.POST,
+        "/oauth/deauthorize",
+        paramsCopy,
+        DeauthorizedAccount.class,
+        options,
+        ApiMode.OAuth);
   }
 
   /**
```

Comment:
nit: I can't comment on the below `getClientId` method but it accesses `Stripe.clientId`, maybe we want to change it to `OAuth.globalResponseGetter.getClientId()` so we keep accesses to global `Stripe` isolated to inside `GlobalStripeResponseGetter`

Reply:
globalResponseGetter doesn't have options exposed publicly (and I don't think it should). I'll take this as a follow up.